Formula/passenger.rb

```diff
@@ -36,9 +52,9 @@ def install
 
     necessary_files = %w[configure Rakefile README.md CONTRIBUTORS
                          CONTRIBUTING.md LICENSE CHANGELOG package.json
-                         passenger.gemspec build bin doc images man dev src
```

Comment:
Because I install the man pages into the proper directory now, at least on my system the old way wasn't working when you actually tried to run `man passenger-config` etc.